WeakAuras/GenericTrigger.lua

```diff
@@ -3941,6 +3941,11 @@ do
     playerMovingFrame:RegisterEvent("PLAYER_STARTED_MOVING");
     playerMovingFrame:RegisterEvent("PLAYER_STOPPED_MOVING");
     playerMovingFrame:SetScript("OnEvent", PlayerMoveUpdate)
+    playerMovingFrame:SetScript("OnUpdate", function(self)
```

Comment:
If OnUpdate is required i think it's safe to stop listening to PLAYER_STARTED_MOVING and PLAYER_STOPPED_MOVING
And then i guess the channeling event spam workaround is not needed anymore

Reply:
Right, that simplifies the code a lot.